draft-gazdag-x509-shbs.md

```diff
@@ -199,9 +186,10 @@ Certificates conforming to [RFC5280] can convey a public key for any public key
 algorithm. The certificate indicates the algorithm through an algorithm
 identifier. An algorithm identifier consists of an OID and optional parameters.
 
-In this document, we define new OIDs for identifying the different hash-based
-signature algorithms. An additional OID is defined in [RFC8708] and repeated
-here for convenience. For all of the OIDs, the parameters MUST be absent.
+In this document, we define new OIDs for identifying the different stateful
+hash-based signature algorithms. An additional OID is defined in [RFC8708] and
```

Comment:
I was thinking about the fact that we use here "hash-based signature algorithms" whereas we use S-HBS everywhere else. But I think that this is okay here due to the context.